### PR TITLE
Add tests 'Helper card' for catalog pages

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add tests 'Helper card' for catalog pages
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | no tests needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21512)

### CODE REVIEW

**Review by nesrineabdmouleh — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/03_catalog/01_products/03_helperCard.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const productsPage = require('@pages/BO/catalog/products');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_catalog_products_helperCard';
+
+
```
Comment (SimonGrn) [inline tests/UI/campaigns/functional/BO/03_catalog/01_products/03_helperCard.js]: Maybe remove this double line break ?

File: tests/UI/pages/BO/catalog/features/index.js [hunk 2]
```diff
@@ -127,6 +131,41 @@ class Features extends BOBasePage {
...
+  /**
+   * @override
+   * Open help side bar
+   * @param page
+   * @returns {Promise<boolean>}
+   */
+  async openHelpSideBar(page) {
+    await page.click(this.helpCardLink);
+
+    return this.elementVisible(page, this.helpContainterBlock, 2000);
```
Comment (SimonGrn) [inline tests/UI/pages/BO/catalog/features/index.js]:
```suggestion
    return this.elementVisible(page, this.helpContainerBlock, 2000);
```

File: tests/UI/pages/BO/catalog/features/index.js [hunk 3]
```diff
@@ -127,6 +131,41 @@ class Features extends BOBasePage {
...
+  /**
+   * @override
+   * Close help side bar
+   * @param page
+   * @returns {Promise<boolean>}
+   */
+  async closeHelpSideBar(page) {
+    await page.click(this.helpCardLink);
+
+    return this.elementNotVisible(page, this.helpContainterBlock, 2000);
```
Comment (SimonGrn) [inline tests/UI/pages/BO/catalog/features/index.js]:
```suggestion
    return this.elementNotVisible(page, this.helpContainerBlock, 2000);
```

File: tests/UI/pages/BO/catalog/features/index.js [hunk 4]
```diff
@@ -7,6 +7,10 @@ class Features extends BOBasePage {
 
     this.pageTitle = 'Features • ';
 
+    // Help card selectors
+    this.helpCardLink = '#toolbar-nav a.btn-help';
+    this.helpContainterBlock = '#help-container';
```
Comment (SimonGrn) [inline tests/UI/pages/BO/catalog/features/index.js]:
```suggestion
    this.helpContainerBlock = '#help-container';
```

File: tests/UI/pages/BO/catalog/features/index.js (hunk 2 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/catalog/features/index.js]: done

File: tests/UI/campaigns/functional/BO/03_catalog/01_products/03_helperCard.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/functional/BO/03_catalog/01_products/03_helperCard.js]: done

**Review by SimonGrn — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Pull request merged**